src/api/json/catalog.json

```
@@ -545,6 +545,14 @@
         "trunk": "https://raw.githubusercontent.com/WordPress/gutenberg/trunk/schemas/json/block.json"
       }
     },
+    {
+      "name": "Block Protocol Metadata Schema",
```

Schema is redundant here - these are all schemas

```suggestion
      "name": "Block Protocol Metadata",
```